packages/server/src/server.js

```diff
@@ -12,14 +12,14 @@ export {
 /**
  * Creates a connection to a service.
  *
- * @template Service
+ * @template {Record<string, any>} Service
```

Comment:
Hey @hugomrdias I did see these changes in few places, yet I'm not sure why is that necessary. If it is indeed necessary might be a good idea to add a code comment so reading code it would be clear why.